examples/multilang-multi-project/proto/lib/google/protobuf/map_lite_unittest.proto

```
@@ -0,0 +1,108 @@
+// Protocol Buffers - Google's data interchange format
+// Copyright 2008 Google Inc.  All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+edition = "2023";
+
+package proto2_unittest;
+
+import "google/protobuf/unittest_lite.proto";
```

Buf (1.54.0)

import "google/protobuf/unittest_lite.proto": file does not exist

(COMPILE)